source/content/guides/drupal-9-legacy-project-to-pantheon.md

```diff
@@ -0,0 +1,251 @@
+
+   ```bash
+   export SOURCE=/absolute/path/to/source/site/codebase
+   export DESTINATION=/absolute/path/to/codebase/cloned/from/pantheon
+   ```
+
+## Copy Existing Configuration
+
+Copy any existing configuration from the source sitem and update the source path as needed to match your configuration folder:
+
+  ```bash{promptUser:user}
+  cp -r $SOURCE/sites/default/files/config/sync/* $DESTINATION/config/
+  # From $DESTINATION:
+  git add config
+  git commit -m "Pull in configuration from source site"
+  ```
+
+If no files are copied through this step, that's acceptable.
+
+## Add Contributed and Custom Code
+
+In your new project structure, replicate your selection of contributed modules and themes, and any custom modules or themes your development team has created.
+
+### Contributed Code
+
+The goal of this process is to have Composer manage all the site's contrib modules, contrib themes, core upgrades, and libraries (referred to as _contributed code_). The only things that should be migrated from the existing site are custom code, custom themes, and custom modules that are specific to the existing site.
+
+
+#### Modules and Themes
+
+
+The steps here ensure that any modules and themes from [drupal.org](https://drupal.org) are in the `composer.json` `require` list.
+
+Once Composer is aware of all the contributed code, you'll be able to run `composer update` from within the directory to have Composer upgrade all the contributed code automatically.
+
+Begin by reviewing the existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`.
+
+1. Review the site and make a list of exactly what versions of modules and themes you depend on. One way to do this is to run the `pm:list` Drush command from within a contributed modules folder (e.g. `/modules`, `/themes`, `/themes/contrib`, `/sites/all/themes`, `/sites/all/themes/contrib`, etc.).
+
+  This will list each module followed by the version of that module that is installed:
+
+  ```bash{promptUser:user}
+  drush pm:list --no-core --fields=name,version  --format=table
+  ```
+
+1. Add these modules to your new codebase using Composer by running the following for each module in the `$DESTINATION` directory:
+
+  ```bash{promptUser:user}
+  composer require drupal/MODULE_NAME:^VERSION
+  ```
+
+  Where `MODULE_NAME` is the machine name of the module in question, and `VERSION` is the version of that module the site is currently using. Composer may pull in a newer version than what you specify, depending upon what versions are available. You can read more about the caret (`^`) in the [Composer documentation](https://getcomposer.org/doc/articles/versions.md#caret-version-range-).
```

Comment:
This should not be the module name; it should be the project name from the "project" field.

Reply:
This is also wrong on https://pantheon.io/docs/guides/composer-convert; I thought that we had corrected it in the past, but the project / module confusion has returned in the latest version.